Tests/scripts/prepare_content_packs_for_testing.sh

```diff
@@ -45,13 +46,18 @@ if [ ! -n "${NIGHTLY}" ]; then
       echo "Did not get content packs to update in the bucket."
     else
       echo "Updating the following content packs: $CONTENT_PACKS_TO_INSTALL ..."
-      python3 ./Tests/Marketplace/upload_packs.py -a $PACK_ARTIFACTS -d $CIRCLE_ARTIFACTS/packs_dependencies.json -e $EXTRACT_FOLDER -b $GCS_BUILD_BUCKET -s $KF -n $CIRCLE_BUILD_NUM -p $CONTENT_PACKS_TO_INSTALL -o -sb $TARGET_PATH -k $PACK_SIGNING_KEY -rt false --id_set_path $ID_SET
+      python3 ./Tests/Marketplace/upload_packs.py -a $PACK_ARTIFACTS -d $CIRCLE_ARTIFACTS/packs_dependencies.json -e $EXTRACT_FOLDER -b $GCS_BUILD_BUCKET -s $KF -n $CIRCLE_BUILD_NUM -p $CONTENT_PACKS_TO_INSTALL -o true -sb $TARGET_PATH -k $PACK_SIGNING_KEY -rt false --id_set_path $ID_SET -pb 'marketplace-dist-private'
       echo "Finished updating content packs successfully."
     fi
   fi
 else
-  echo "Updating all content packs for nightly build..."
-  python3 ./Tests/Marketplace/upload_packs.py -a $PACK_ARTIFACTS -d $CIRCLE_ARTIFACTS/packs_dependencies.json -e $EXTRACT_FOLDER -b $GCS_BUILD_BUCKET -s $KF -n $CIRCLE_BUILD_NUM -o -sb $TARGET_PATH -k $PACK_SIGNING_KEY -rt false --id_set_path $ID_SET
+  if [ -n "${NIGHTLY}" ]; then
+    echo "Updating all content packs for nightly build..."
+  elif [ -n "${BUCKET_UPLOAD}" ]; then
+    echo "Updating all content packs for upload packs to production..."
+  fi
+  [ -n "${BUCKET_UPLOAD}" ] && REMOVE_PBS=true && OVERRIDE_ALL_PACKS=false || REMOVE_PBS=false && OVERRIDE_ALL_PACKS=true
```

Comment:
In which cases test playbooks are uploaded?

Reply:
nightly/regular build